src/bin_launcher/Oni2.re

```diff
@@ -40,13 +40,13 @@ let spec =
     (
       "--disable-syntax-highlighting",
       passthrough,
-      "Turn off syntax highlighting.",
+      " Turn off syntax highlighting.",
```

Comment:
Whilst I was in the CI, I fixed this.

If you specify the description with a space at the start, it will be properly aligned with all the rest of the flags.

Reply:
Thanks for cleaning this up, too 👍